### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by Chun-Yang**
@djhi Please review this.
Thank you!

### CODE REVIEW

**Comment by Chun-Yang** [comment]
@djhi I understand that you are busy. But we need to fix some bugs. I will wait for your review. And mean while I have to use my own fork.

**Comment by djhi** [comment]
Hey, I'm going to review it today.

File: README.md [hunk 1]
```diff
@@ -16,11 +13,7 @@ Adds [materialize](http://materializecss.com/) templates for autoform.
 You must add materialize CSS and JavaScript yourself. Some packages can help:
 
 - [materialize:materialize](https://atmospherejs.com/materialize/materialize) `meteor add materialize:materialize`
-- [d0minikk:materialize-meteor](https://atmospherejs.com/d0minikk/materialize-meteor) `meteor add d0minikk:materialize-meteor`
-- [grigio:materialize-sass](https://atmospherejs.com/grigio/materialize-sass) `meteor add grigio:materialize-sass`
-  Note this one doesn't add the scripts but allows you to customize the CSS with SASS
-- [yang2007chun:materialize-scss](https://atmospherejs.com/yang2007chun/materialize-scss) `meteor add yang2007chun:materialize-scss`
-  Note this one allows you to customize with SASS
+- [poetic:materialize-scss](https://atmospherejs.com/poetic/materialize-scss) `meteor add yang2007chun:materialize-scss`
```
Comment (djhi) [inline README.md]: Why removing all the other options ?

File: inputTypes/boolean-radios/boolean-radios.js [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
-        if (this.value === true) {
-            atts.checked = '';
-        }
-        return atts;
-    },
-    dsk: function() {
-        return {
-            'data-schema-key': this.atts['data-schema-key']
-        };
+  attrs: Utility.attsToggleInvalidClass
```
Comment (djhi) [inline inputTypes/boolean-radios/boolean-radios.js]: `attrs` does not seem to be used anywhere in the template

File: inputTypes/boolean-select/boolean-select.js [hunk 3]
```diff
@@ -1,8 +1,9 @@
 Template.afBooleanSelect_materialize.helpers({
-    atts: function() {
-        return Utility.helpers.attsToggleInvalidClass(this.atts);
-    },
-    optionAtts: Utility.helpers.optionAtts
+  attrs: Utility.attsToggleInvalidClass
```
Comment (djhi) [inline inputTypes/boolean-select/boolean-select.js]: Same as previous attrs

File: inputTypes/color/color.js [hunk 4]
```diff
@@ -1,5 +1,3 @@
 Template.afInputColor_materialize.helpers({
-    atts: function() {
-        return Utility.helpers.attsToggleInvalidClass(this.atts);
-    }
+  attrs: Utility.attsToggleInvalidClass
```
Comment (djhi) [inline inputTypes/color/color.js]: Same for all attrs

**Comment by djhi** [comment]
Looks great :+1: 
I'll try to take some time to add tests

File: README.md (hunk 1 above)
Comment (Chun-Yang) [inline README.md]: The two others are flagged.

**Comment by Chun-Yang** [comment]
I did more refactoring. Tell me if you have questions. :smile:

**Pull request merged**